lib/dockerfile/instruction-parser.ts

```diff
@@ -54,7 +55,10 @@ function getPackagesFromRunInstructions(runInstructions: string[]) {
 
         packages.forEach((pkg) => {
           // Use package name without version as the key
-          const name = pkg.split("=")[0];
+          let name = pkg.split("=")[0];
+          if (name.slice(0, 1) === "$") {
```

Comment:
isn't `name.slice(0,1)` equal to just `name[0]`? might be more readable, not critical

Reply:
or `name.startsWith("$")`?